src/api-endpoints.ts

```diff
@@ -1,10 +1,12 @@
+import { BASE_URL as DEFAULT_BASE_URL, UFC_ENDPOINT } from './constants';
 import { ISdkParams } from './http-client';
 
-const UFC_ENDPOINT = '/flag-config/v1/config';
-
 /** Utility class for constructing an Eppo API endpoint URL given a provided baseUrl and query parameters */
 export default class ApiEndpoints {
-  constructor(private readonly baseUrl: string, private readonly queryParams: ISdkParams) {}
+  constructor(
+    private readonly baseUrl: string = DEFAULT_BASE_URL,
+    private readonly queryParams: ISdkParams,
+  ) {}
```

Comment:
Also I moved the setting of `baseUrl` with `DEFAULT_BASE_URL` into the constructor because `ApiEndpoints` needs this information to form the URL. Alternatively, `BASE_URL` could be separately added to the exports in `index.ts` and then passed in as an argument when creating an instance of `ApiEndpoints` in js-client-sdk